spec/exp_descriptor_en_spec.rb

```diff
@@ -375,7 +375,21 @@ def desc(expression, opts = {})
       end
 
       it 'year increments' do
-        expect(desc('0 0 0 1 MAR * 2010/5')).to eq('At 0:00 AM, on day 1 of the month, only in March, every 5 years, starting in 2010')
+        expect(desc('0 0 0 1 MAR * 2010/5')).to eq('At 12:00 AM, on day 1 of the month, only in March, every 5 years, starting in 2010')
+      end
+    end
+
+    context 'timezone' do
+      it 'minute span' do
+        tz = TZInfo::Timezone.get('America/Los_Angeles')
+        hour = tz.period_for_local(Time.now).zone_identifier.to_s == 'PDT' ? 4 : 3
+        expect(desc('0-10 11 * * *', {}, 'America/Los_Angeles')).to eq("Every minute between #{hour}:00 AM and #{hour}:10 AM")
+      end
+
+      it 'twice a day' do
+        tz = TZInfo::Timezone.get('America/Los_Angeles')
+        hour = tz.period_for_local(Time.now).zone_identifier.to_s == 'PDT' ? 5 : 4
+        expect(desc('0 0,12 * * *', {}, 'America/Los_Angeles')).to eq("At #{hour}:00 PM and #{hour}:00 AM")
```

Comment:
would it make sense to have a test with a timezone ahead of GMT (e.g.  Eastern Europe)?

Reply:
I verified all the timezone tests by changing my system clock and timezone. Moving to standard time returns the expected results.